src/hooks/useIsMounted.ts

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react'
+
+export function useIsMounted() {
+  const [mounted, setMounted] = React.useState(false)
+
+  React.useEffect(() => setMounted(true), [])
+
+  return mounted
+}
```

Comment:
This is a simple hook to check to see if the component is mounted. 

When using WAGMI hooks often times you will get hydration errors because of the hooks. 
https://nextjs.org/docs/messages/react-hydration-error